zebedee-reader/src/main/java/com/github/onsdigital/zebedee/reader/api/ReadRequestHandler.java

```diff
@@ -74,6 +74,32 @@ public Content find(HttpServletRequest request, DataFilter dataFilter, String ur
         }
     }
 
+    /**
+     * Finds requested published content
+     * <p>
+     * If requested uri ends in "latest" it will return latest edition of bulletin or article content, throws BadRequestException given uri is not a bulletin or article content
+     *
+     * @param request
+     * @param dataFilter
+     * @return Content
+     * @throws ZebedeeException
+     * @throws IOException
+     */
+    public Content findPublishedContent(HttpServletRequest request, DataFilter dataFilter) throws ZebedeeException, IOException {
+        String uri = extractUri(request);
+        return findPublished(request, dataFilter, uri);
+    }
```

Comment:
@daiLlew  The `findPublishedContent(…)` and `findPublished(…)` methods are essentially copies of the `findContent(…)` and `find(…)` methods respectively that don't have the collectionID resolution step (see line 67 above). You'll notice that when `getLatestContent(…)` or `getContent(…)` is called the second argument which is the collectionID is hard coded to null to ensure it can't be used.